app/views/components/input_radio.scala.html

```diff
@@ -20,13 +20,14 @@
         field: Field,
         legend: String,
         legendClass: Option[String] = None,
+        fieldSetClass: Option[String] = None,
         hint: Option[String] = None,
         trackGa: Boolean = false,
         inputs: Seq[RadioOption]
 )(implicit messages: Messages)
 
 <div class="form-group @if(field.hasErrors){form-group-error}">
-    <fieldset id="@{field.id}">
+    <fieldset @if(fieldSetClass.nonEmpty){class ="@fieldSetClass.get}" id="@{field.id}">
```

Comment:
Check if the quote should be before the bracket ... ie:
{class = "@fieldSetClass.get"}